Sources/AccioKit/Services/XcodeProjectIntegrationService.swift

```diff
@@ -244,7 +244,7 @@ final class XcodeProjectIntegrationService {
     }
 
     private func verifyBundleVersion(of product: FrameworkProduct) throws {
-        let plistURL = product.frameworkDirUrl.appendingPathComponent("Info.plist")
+        let plistURL = product.frameworkDirUrl.appendingPathComponent("Resources").appendingPathComponent("Info.plist")
```

Comment:
This can be simplified to `appendingPathComponent("Resources/Info.plist")`.

Reply:
While that is true both `Resources` and `Info.plist` are individual path components as by the definition of the Apple API as in https://developer.apple.com/documentation/foundation/nsstring/1414489-pathcomponents 

While in this case irrelevant it manually bakes in the path separator.
I would consider it bad practice - but no problem to change this.